codec/src/main/java/io/netty/handler/codec/CharSequenceValueConverter.java

```diff
@@ -126,12 +124,8 @@ public CharSequence convertTimeMillis(long value) {
 
     @Override
     public long convertToTimeMillis(CharSequence value) {
-        try {
-            return HeaderDateFormat.get().parse(value.toString());
-        } catch (ParseException e) {
-            PlatformDependent.throwException(e);
-            return 0;
-        }
+        Date date = HeaderDateFormatter.parse(value);
```

Comment:
I guess so, fixing. Then, I think this method should stop throwing in a future version when it's possible to break compatibility.